### ISSUE TITLE
Location name cut off after slash

### ISSUE THREAD

**Issue by PhantomWatson** [BallStateCBER/muncie_events#27]
"Collective" is cut off from the location name "Chameleon Salon/Collective" in http://muncieevents.com/location/Chameleon%20Salon%2FCollective.

**Comment by ericadeefox** [comment]
It's not accepting %2F as a URI character. Sometimes apache rejects %2F & %5C (which is the \ slash) but it weirdly accepts %5C... and every other URI character I tried.

More on this: https://jira.spring.io/browse/SPR-11101

**Comment by ericadeefox** [comment]
Yeah, it's treating all the URI characters literally, e.g. if you type in a space it automatically changes to %20. Therefore if you type in %2F, the browser treats it like a forward slash... e.g. http://muncieevents.com/location/The%20Cup%2Fpast takes you to the past events page.

**Comment by ericadeefox** [comment]
Happening in multiple browsers so it's not a browser thing.....
